packages/tenant-process/package.json

```diff
@@ -5,6 +5,8 @@
   "main": "dist",
   "type": "module",
   "scripts": {
+    "test": "vitest",
```

Comment:
still useful?

Reply:
We could keep both the scripts until we decide whether to add more tests, e.g. for authorization.

Reply:
Update: the `test:it` script has been removed.